### ISSUE TITLE
Use Cobigen to switch between frameworks

### ISSUE THREAD

**Issue by MikeSchumacherCapgemini** [devonfw/cobigen#1384]
As a Cobigen user, I may want to easily swap between different frameworks or similar using Cobigen so that i can save time by not doing it myself.

An example for this would be to usually use Spring Boot to launch applications, but wanting to test out another type of framwork (e.g. JBoss)

This is a workshop topic, any ideas are welcome and can be discussed in the comments

**Comment by hohwille** [comment]
I think this would be nice but is extremely complex so cost vs. usage might not be in balance.
However, we could lower the expectation: If we do not expect CobiGen to magically transform any spring-boot app into quarkus, JBoss, micronaut and vice-versa we could consider all the aspects of this problem that we can solve easiy so we can take quite some effort but not all for doing so.
In the end I still think that this use-cases is extremely rare. I have never seen any project doing that on a regular path. Only for innovation and migration projects I have seen some upgrades from technology X to Y. But as X and Y could be hardly anything and every combination would mean a hugh effort, I personally see many other stories we should do first with our capacities.

**Comment by MikeSchumacherCapgemini** [comment]
I completely agree that this sounds like too much work for what we would achieve.
When assuming that you want the same data from an input file for different frameworks, it might be possible to extract it from generated spring-boot files, then delete those files and generate quarkus instead.

This is assuming that we can generate an amount of different frameworks to have this feature do enough, in addition to being able to extract the data out of generated files OR for Cobigen to know what data was used when originally generating.

But then another question is, is it worth it to use a possibly "unstable" feature like this instead of making the user delete the old files and generating new ones.
